.github/workflows/CI.yml

```diff
@@ -38,6 +38,16 @@ jobs:
         with:
           submodules: true
 
+      # Git functionality in CMAKE file does not work with given ubuntu image. Therefore, handling it here.
```

Comment:
What is meant by this? If we do this, everytime we add a patch we will need to update this.

Reply:
Good Idea. Let me do that.

Reply:
Correct. Git command in CMAKE file like `git submodule update --init --` or `git apply` does not work for ubuntu. Need to talk with @peterzhuamazon to see if we can fix it by using different ubuntu image.

Reply:
@heemin32 what was the conclusion for this?

Reply:
No update yet. Need to talk with @peterzhuamazon once he come back from a vacation. The code is needed as of now to make it work though.

Reply:
is that discussion blocking this PR?

Reply:
It won't.